google/cloud/sql/connector/instance_connection_manager.py

```diff
@@ -365,26 +366,14 @@ async def _force_refresh(self) -> bool:
             self._next.cancel()
             # schedule a refresh immediately with no delay
             self._next = self._loop.create_task(self._schedule_refresh(0))
+            await asyncio.wait_for(self._next, timeout)
```

Comment:
I think we already talked about this too, but this should be a instant sync operation. It need to trigger the next refresh, but tot await on the results.